specification/api-correlationcontext.md

```diff
@@ -1,24 +1,27 @@
-# DistributedContext API
+# Correlations API
 
-An `Entry` is used to label anything that is associated
-with a specific operation, such as an HTTP request.
+An `Entry` is used to label anything that is associated with a specific
+operation, such as an HTTP request.
 
-`DistributedContext` is an abstract data type that represents a collection of entries.
-Each key of `DistributedContext` is associated with exactly one value. `DistributedContext` is serializable,
-to facilitate propagating it not only inside the process but also across process boundaries.
-`DistributedContext` is used to annotate telemetry with the name:value pair `Entry`.
-Those values can be used to add dimension to the metric or additional context properties to logs and traces.
+`CorrelationContext` is an abstract data type that represents a collection of
+entries. Each key of `CorrelationContext` is associated with exactly one value.
+`CorrelationContext` is serializable, to facilitate propagating it not only
```

Comment:
I'd rewrite this sentence, something like "`CorrelationContext` is serialized using the W3C Correlation Context specification." (https://w3c.github.io/correlation-context/).

(I don't see why it matters that it's serializable for propagation inside the process.)

Reply:
Updated